Resources/Prototypes/Entities/Objects/Consumable/drinks_cups.yml

```diff
@@ -9,6 +9,12 @@
     solutions:
       drink:
         maxVol: 10
+  - type: FitsInDispenser
+    solution: drink
+  - type: DrawableSolution
+    solution: drink
+  - type: InjectableSolution
```

Comment:
Again a bit sus. If I fire chem ammo at a cup, should the chem ammo inject into it?

Reply:
I'll swap it out for refillable and make InjectorComponent able to use that (as if the injector was being used above the cup, similar to an eyedropper)